### ISSUE TITLE
Ledger - Txn version not supported

### ISSUE THREAD

**Issue by dearkland** [Zondax/ledger-polkadot#127]
I'm not able to make any transactions with my Ledger. Tried sending, bonding, etc. Tried on Ledger live, and through polkadot.js and I keep getting the same error. 

Txn version not supported

Reached out in Discord and Ledger support and I get he same responses.. Is everything up-to-date? YES.  

Firmware - current
Ledger Live - current

Am I missing something or is there a problem? I see others having the same problem in the discord.  Any help would be GREATLY appreciated.

Thanks!

**Comment by Sikolopo** [comment]
> 

they ask you to update the application to Polkadot 11.9170.x
but there is no such update in Ledger (I have the maximum 10.9160
leger app downloaded from the website ( latest update )

**Comment by Sikolopo** [comment]
Also when I try to restore access to the Dot through the phrase for example on the website js / trust wallet. it gives me a different address without money )))) I am now afraid to hard reset

**Comment by jleni** [comment]
We released v11.9170 16 days ago

**Comment by jleni** [comment]
> Also when I try to restore access to the Dot through the phrase for example on the website js / trust wallet. it gives me a different address without money )))) I am now afraid to hard reset

BIP39 Mnenomics (Ledger) are not processed correctly by polkadot.js. 

Substrate treats mnemonic in a ways that is different from other chains. Given a mnemonic, polkadot.js will generate a different address but this does not mean that funds in your Ledger BIP39 have been compromised in any way.

**Comment by Sikolopo** [comment]
I checked on the ledger nano s - I installed the new update on it 
But the nano x still has the same problem - I am not prompted to install the new update

**Comment by jleni** [comment]
Thanks for the info! We are trying to reach Ledger to understand why Nano X version was not released. The codebase is shared so it is hard to understand why one version is available and the other is not.

**Comment by Sikolopo** [comment]
> BIP39 Mnenomics (Ledger) are not processed correctly by polkadot.js.
> 
> Substrate treats mnemonic in a ways that is different from other chains. Given a mnemonic, polkadot.js will generate a different address but this does not mean that funds in your Ledger BIP39 have been compromised in any way.

but a question. how can the Ledger mnemonics be used to retrieve a dot in words? if, for example, the Ledger is closed

**Comment by jleni** [comment]
No, this should not be the case.

We created a library to do this.. but runs in software so we don't recommend using it unless really necessary.

I explained the reasons and our approach here
https://github.com/Zondax/ledger-kusama/issues/129#issuecomment-1065970257

TLDR: Ledger uses BIP39 mnemonics (bitcoin standard) and Substrate decided to use something different that is substrate specific. 

We wrote code to allow BIP39 mnemonics to be used as substrate keys but it has not been integrated anywhere. The code and methodology is there publicly. 

Given that we have received this question multiple times, we will make this a bit more public and explicit.

**Comment by jleni** [comment]
The main reason we don't recommend using this tool is because most people that have a Ledger device share the same mnemonic for all the funds they have in the device. So when using a software tool without Ledger mnemonic you may be compromising the security of other coins that you have in the device.

**Comment by Sikolopo** [comment]
> The main reason we don't recommend using this tool is because most people that have a Ledger device share the same mnemonic for all the funds they have in the device. So when using a software tool without Ledger mnemonic you may be compromising the security of other coins that you have in the device.

as you wrote in that thread it would be very convenient that it would be an online tool for quick extraction) because other coins were successfully added to a third-party application and successfully withdrawn them to another cold wallet, except dots Q_Q

**Comment by Sikolopo** [comment]
> > The main reason we don't recommend using this tool is because most people that have a Ledger device share the same mnemonic for all the funds they have in the device. So when using a software tool without Ledger mnemonic you may be compromising the security of other coins that you have in the device.
> 
> as you wrote in that thread it would be very convenient that it would be an online tool for quick extraction) because other coins were successfully added to a third-party application and successfully withdrawn them to another cold wallet, except dots Q_Q

If this is already available, I would use it to translate my Dots

**Comment by mezhcoder** [comment]
If you need to make an urgent transfer, is it safe for you to Build a new version on your Ledger?

**Comment by Sikolopo** [comment]
> If you need to make an urgent transfer, is it safe for you to Build a new version on your Ledger?

my LedgerLive has the newest update, but it won't let me update my DOT to the new version

**Comment by mezhcoder** [comment]
> my LedgerLive has the newest update, but it won't let me update my DOT to the new version

I have the same problem. I just want to know if I Build on GitHub a new version of the Polkadot application, will my funds be safe or some bug might happen

**Comment by Sikolopo** [comment]
> I have the same problem. I just want to know if I Build on GitHub a new version of the Polkadot application, will my funds be safe or some bug might happen

Yesterday on miningclub the man did it, and he did well, but it is necessary to understand the bases and ubuntu))) still I can not do it

**Comment by jleni** [comment]
Ledger does not allow side loading software in nano x. Only in nano S, but that device already has the latest version in Live.

We already provide an installer for test purposes on every release but you would need a nano S or S+

**Comment by Sikolopo** [comment]
> Ledger does not allow side loading software in nano x. Only in nano S, but that device already has the latest version in Live.
> 
> We already provide an installer for test purposes on every release but you would need a nano S or S+

 just wait for an update ? or can I convert my Ledger phrases online and restore them to js

**Comment by jleni** [comment]
They have the code and merged in github already, so I guess it is something in their infra that resulted in the binary not being exposed..

About the script, I can ask someone in my team to start working on this ASAP, however, I personally find it very risky and can result in people use this to scam non-technical users, etc..

Just to understand a bit better the situation. You only have a Nano X device?
